aten/src/TH/generic/THTensor.cpp

```diff
@@ -58,9 +58,10 @@ scalar_t *THTensor_(data)(const THTensor *self) {
 THTensor *THTensor_(new)(void)
 {
   return c10::make_intrusive<at::TensorImpl, at::UndefinedTensorImpl>(
-    c10::intrusive_ptr<at::StorageImpl>::reclaim(THStorage_(new)()),
-    at::DispatchKey::CPU
-  ).release();
+             c10::intrusive_ptr<at::StorageImpl>::reclaim(THStorage_(new)()),
+             at::DispatchKey::CPU,
+             caffe2::TypeMeta::Make<scalar_t>())
```

Comment:
I think this should be `quanitzed_t` when `THQUANTIZED` is defined, as was done in `generic/THStorage.cpp`